code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1360,8 +1360,10 @@
 		dna.species.create_organs(src)
 
 	for(var/obj/item/thing in kept_items)
-		equip_to_slot_if_possible(thing, kept_items[thing])
+		var/equiped = equip_to_slot_if_possible(thing, kept_items[thing])
```

Comment:
equiped -> equipped
😃

Reply:
imaginne not writting a sugestioon for por old mee